src/core/login/keys.js

```diff
@@ -318,9 +318,15 @@ export async function createCurrencyWallet(
     if (tools.importPrivateKey == null) {
       throw new Error('This wallet does not support importing keys')
     }
-    keys = await tools.importPrivateKey(opts.importText, opts.keyOptions)
+    keys = {
+      ...(await tools.importPrivateKey(opts.importText, opts.keyOptions)),
+      imported: true
+    }
   } else {
-    keys = await tools.createPrivateKey(walletType, opts.keyOptions)
+    keys = {
+      ...(await tools.createPrivateKey(walletType, opts.keyOptions)),
+      imported: true
```

Comment:
This one needs to be false.

Reply:
Oops. That was a derp on my part. Fixed up.